tests/rptest/services/redpanda_installer.py

```diff
@@ -21,18 +22,26 @@
 
 
 def wait_for_num_versions(redpanda, num_versions):
+    # Use a single node so the metadata about brokers have a consistent source
+    # in case we retry.
+    node = redpanda.nodes[0]
+
     def get_unique_versions():
-        node = redpanda.nodes[0]
-        brokers_list = \
-            str(node.account.ssh_output(f"{redpanda.find_binary('rpk')} redpanda admin brokers list"))
+        try:
+            brokers_list = \
+                json.dumps(redpanda._admin.get_brokers(node=node))
+        except Exception as e:
+            redpanda.logger.debug(f"Failed to list brokers: {e}")
```

Comment:
Oh oops didn't see this before merging. Fair point, it's probably redundant for admin requests, though it might be worth catching and logging still in case there's some weird stuff in the json.